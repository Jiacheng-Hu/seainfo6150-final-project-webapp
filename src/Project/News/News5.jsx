import React from "react";
import HTMLText from "../HTMLText/HTMLText";

const News5 = props => {

    return (
        <article className="container">
            <header>
                <h1>{props.newsItem[4].title}<br></br> <small>by {props.newsItem[4].author}</small></h1>
                <time dateTime={props.newsItem[4].timeStamp}></time>
            </header>
            <div className="news">
                <img className="news-image-item" src={props.newsItem[4].image._url} alt="news5"/>
                <HTMLText format={props.newsItem[4].text}/>
            </div>
        </article>
    )
}

export default News5;