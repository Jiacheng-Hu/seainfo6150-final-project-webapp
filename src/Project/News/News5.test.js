import React from "react";
import News5 from "./News5";


const newsItem = [
    {
        slug: "news1",
        title: "How bike-friendly 'slow streets' are changing cities",
        shortText: "Covid-19 caused rapid shifts in travel infrastructure to make streets friendlier to bikers and pedestrians. Can these changes endure as people return to their cars and public transport?",
        text: "<p>Moving around Bogotá can be a bit of a Jekyll-or-Hyde experience. On one hand, the city is infamous for having the world’s worst traffic. Yet, on the other, its cycling infrastructure is considered an exemplary model of sustainable urban mobility, according to the Copenhagenize Index, which ranks bike-friendly cities. The Colombian capital even spawned a now-international movement in the 1970s called Ciclovía, which sees 1.5 million people pedal across 128km (80 miles) of car-free streets each Sunday morning.</p><p>So, when the pandemic reached its shores in mid-March, Bogotá Mayor Claudia López, an avid cyclist herself, enacted one of the world’s first plans to encourage bike travel, using traffic cones to create 76km of temporary lanes.</p><p>“Everyone started using a bicycle, and they already knew how to get around on one because we have this bike culture thanks to the Ciclovía,” says Carlos Pardo, a local cycling advocate and senior advisor at the New Urban Mobility Alliance. Pardo got involved at the outset of the pandemic by partnering with a local bikeshare company to provide 400 free e-bikes to health workers. Now, he’s busy persuading the public that the government’s new bike lanes should become permanent fixtures.</p><p>“Some drivers say, ‘you took away our lane’, but we’re saying, we took one car lane and made a two-lane bidirectional bike lane,” he explains. “So, you’re duplicating the effectiveness of the space, and moving more people per hour, per direction.” Thanks to Pardo’s advocacy as well as quick actions by Bogotá’s mobility secretariat, the extent of the new bike lanes is now 84km, bringing the extensive cycle network to 634km in total. The government says at least 65km of the new lanes will become permanent.</p><p>Biking has enjoyed a renaissance around the world as urbanites shun public transport for the relative safety of a two-wheeled commute. Now, many advocates like Pardo are working with local governments in the hope of turning these pandemic-response measures into lasting changes – ones that are more plausible now than ever after lockdowns provided an unprecedented opportunity to fast-track infrastructure trials. The results of these urban planning experiments could not only radically alter the way we commute across global cities, but also make them more resilient to future shocks.</p><p>Tabitha Combs, a lecturer at the University of North Carolina at Chapel Hill, has been crowdsourcing data on local actions affecting walking and cycling during the pandemic. She’s identified at least 365 global cities that have allocated new street space for these activities since data-gathering began on 24 March.</p>",
        pubDate: "Thu, 12 Nov 2020 15:12:24 GMT",
        author: "Mark Johanson",
        image: {
            _url: "https://ychef.files.bbci.co.uk/1600x900/p08y7xg0.webp",
            _height: "433",
            _width: "770"
        },
        timeStamp: "2020-11-12T15:12:24.000Z",
        displayDate: "November 12th 2020"
    },
    {
        slug: "news2",
        title: "What happens to workplace perks when no-one's in the office?",
        shortText: "Drinks on tap, rock-climbing walls, gourmet cafeterias – perks are a hallmark of many desirable workplaces. But when workers are not in the office to actually enjoy them, companies are being forced to reevaluate their incentives and work culture.",
        text: "<p>Greg Domino remembers the majestic views of New York City from his former company’s office on the seventy-second floor of a Financial District skyscraper. The view was one of many perks that the sports-streaming service afforded to office workers like him. Domino, a 33-year-old PR professional, recalls “these gorgeous panoramic views of New York, ping-pong tables, kombucha, often times breakfast”. Employees' Instagram Stories were flooded with images of their office trappings.</p><p>But when the office temporarily closed eight months ago due to the pandemic, Domino wasn’t wistful about losing his daily dose of corporate culture, the view or free kombucha. He relished working on simpler terms from his house for the first few months. And since being made redundant by the company in September, he’s realised he doesn’t want the fancy office back at all. “As the father of two small kids with a wife who also works,” he says, “the flexibility of being home far outweighs [the office perks]”.</p><p>Over the past several years, more and more companies have been offering flashy amenities in the hope that it will help them compete for top talent. But with many corporate employees working from home since the start of the Covid-19 lockdowns, it’s unclear what purpose office spaces with ping-pong tables, espresso bars, meditation rooms and other enticing perks serve when there aren’t any employees in the building to use them.</p><p>The souped-up workspace has its roots in Silicon Valley, and the dawn of the US technology industry. In the first half of the 20th Century, fledgling technology firms filled their offices with luxuries as a means of luring promising recruits.</p><p>“[Hewlett-Packard], founded [in] 1939, was the first born-in-the-Valley company to incorporate these perks,” according to Margaret O'Mara, a history professor at the University of Washington. “[They] included not only outdoor patios and recreational areas, but also stock options in a growing company. This set-up got the ball rolling, and the industry’s next wave of firms laid the foundation for what we know as today’s start-up vibe. “HP set the tone for the many companies that followed,” says O’Mara, “who took it to the next level in the 70s and 80s with jogging paths and swimming pools, and in the 90s and 2000s with free food and climbing walls and ping-pong tables.”</p><p>In more recent years, Silicon Valley giants and smaller firms alike have outfitted their offices with gyms, cocktail lounges and even a music rehearsal space and an Oval Office replica. Beyond being a recruitment incentive, amenities like this are meant to foster a palpable sense of company culture that sets certain companies apart from the herd.</p>",
        pubDate: "Mon, 16 Nov 2020 13:19:10 GMT",
        author: "Sam Blum",
        image: {
            _url: "https://ychef.files.bbci.co.uk/1600x900/p08y16jh.webp",
            _height: "433",
            _width: "770"
        },
        timeStamp: "2020-11-16T13:19:10.000Z",
        displayDate: "November 16th 2020"
    },
    {
        slug: "news3",
        title: "How can employees also be social-media activists?",
        shortText: "Experts say that unfiltered social activism could cost you valuable career currency. What should workers who want to raise their voices do?",
        text: "<p>If, earlier this year, you’d asked Jarred Harewood about his social media habits, the 32-year-old Brooklyn-based software developer would have described himself as “very, very careful” about what he posted. To avoid alienating friends and colleagues, he didn’t talk politics on his feed, but instead kept it light with snapshots of food and travel. But he says being careful isn’t an option anymore, especially as a black American.</p><p>The deaths of George Floyd, Breonna Taylor, Daniel Prude and many other black people at the hands of police have recently made him “significantly more vocal and outspoken” on Instagram. In his stories, he posts about Black Lives Matter and issues at the centre of the recent US election, such as health care. He knows it’s a fine line to walk these days. “You could lose your job for these views and you may not have a lot of options out there,” says Harewood. “Each person has to weigh that themselves.”</p><p>As huge organisations such as Amazon, Netflix, the Pokémon Company, pharmaceutical giant Novartis and the US’s National Football League have publicly expressed support for social justice issues, an increasing number of people have been empowered to raise their voices in their own workplaces. And there’s lots to talk about: inequities at school and work as well as views about issues such as politics, public health, the environment, privilege and bigotry.</p><p>Of course, workers have been activists for years – but it used to be the case that if you were rallying over the weekend, your colleagues and boss could be none the wiser on Monday. Plus, the relationship people expect to have with their workplace has changed over the generations. While baby boomers were traditionally motivated by company loyalty and duty, younger workers place a high value on authenticity and are civic minded. They may be less willing to mute their beliefs.</p><p>So, now, with so much activism shifting to social media for so many, it’s not so easy to delineate activist from worker bee. And, unfortunately, experts say that being unfiltered could cost you valuable career currency. According to a growing area of research, social-media activism and gainful employment may be at odds.</p><p>What’s a worker who wants to raise their voice to do?</p><p>It's a tricky line to walk. In a study published earlier this year, researchers at Pennsylvania State University (Penn State) found that “opinionatedness” on social media negatively affected perceptions of job suitability among recruiters. In the experiment, hundreds of hiring managers were shown the same Facebook photo of a simulated job candidate. The subject stood in front of an American flag with one of two captions: either a benign comment referring to a “beautiful day”, or a more lengthy, opinionated version about politics and voting. It ended with, “My voice will be heard. Will yours?” While the photos and corresponding credentials were identical, evaluations were lower for the candidates with the political caption.</p>",
        pubDate: "Fri, 27 Nov 2020 17:10:14 GMT",
        author: "Zulekha Nathoo",
        image: {
            _url: "https://ychef.files.bbci.co.uk/1600x900/p08yvvy7.webp",
            _height: "433",
            _width: "770"
        },
        timeStamp: "2020-11-27T17:10:14.000Z",
        displayDate: "November 27th 2020"
    },
    {
        slug: "news4",
        title: "The psychology behind ‘revenge bedtime procrastination’",
        shortText: "Many young Chinese workers prioritise leisure time over sleep after long work days – even though they know it’s unhealthy. What’s driving this behaviour?",
        text: "<p>Emma Rao spent almost three years on China’s notorious ‘996 schedule’: working from nine in the morning to nine in the evening, six days a week. Rao, who is originally from Nanjing, moved to financial hub Shanghai about five years ago to work for a multinational pharmaceutical company. The job quickly took over her life.</p><p>“I was almost depressed,” she says. “I was deprived of all my personal life.” After her shift, which sometimes included overtime, she had a small window to eat, shower and go to bed – but she sacrificed sleep to eke out some personal time. Often, Rao would stay up surfing the internet, reading the news and watching online videos until well after midnight.</p><p>Rao was doing what the Chinese have called ‘bàofùxìng áoyè’ – or ‘revenge bedtime procrastination’. The phrase, which could also be translated as ‘retaliatory staying up late’, spread rapidly on Twitter in June after a post by journalist Daphne K Lee. She described the phenomenon as when “people who don’t have much control over their daytime life refuse to sleep early in order to regain some sense of freedom during late-night hours”.</p><p>Her post clearly struck a chord. In a reply ‘liked’ more than 4,500 times, Twitter user Kenneth Kwok tweeted: “Typical 8 to 8 in office, [by the time I] arrive home after dinner and shower it’s 10 p.m., probably won’t just go to sleep and repeat the same routine. A few hours of ‘own time’ is necessary to survive.”</p><p>It’s not clear exactly where this term came from. The earliest mention this reporter found was in a blog post dated November 2018, although its origins likely predate this. The post’s author – a man from Guangdong province – wrote that during the workday he “belonged to someone else,” and that he could only “find himself” when he got home and could lie down. This revenge bedtime procrastination was sad, he wrote, because his health suffered, but it was also “great” because he got a bit of freedom.</p><p>The phrase might have been popularised in China, but the phenomenon it describes is likely widespread, with over-stretched workers all over the world putting off bedtime to claim some precious personal time – even though they know it’s not good for them.</p><p>Experts have long warned that insufficient sleep is an unheeded global public-health epidemic. The 2019 Phillips Global Sleep Survey, which received more than 11,000 responses from 12 countries, showed that 62% of adults worldwide feel they don’t get enough sleep, averaging 6.8 hours on a weeknight compared to the recommended amount of eight hours. People cited various reasons for this shortfall, including stress and their sleeping environment, but 37% blamed their hectic work or school schedule.</p><p>In China, a national survey in 2018 showed that 60% of people born after 1990 were not getting enough sleep, and that those living in the biggest cities suffered the most. The tech companies who created 996 culture tend to be based in big cities, and their work practices have influenced other sectors. A recent report by state broadcaster CCTV and the National Bureau of Statistics said the average Chinese employee only had 2.42 hours per day when they were not at work or asleep, down by 25 minutes from the previous year.</p>",
        pubDate: "Thu, 26 Nov 2020 10:25:45 GMT",
        author: "Lu-Hai Liang",
        image: {
            _url: "https://ychef.files.bbci.co.uk/1600x900/p08zf427.webp",
            _height: "433",
            _width: "770"
        },
        timeStamp: "2020-11-26T10:25:45.000Z",
        displayDate: "November 26th 2020"
    },
    {
        slug: "news5",
        title: "Coronavirus: Why dating feel so different now",
        shortText: "Research shows that a potential health threat can transform the way we think about and approach romantic interactions.",
        text: "<p>Emily, a 29-year-old surveyor from London, says she’s always been something of an introvert. She’d dabbled in dating, but when the UK’s first lockdown restrictions ended in July, she was reluctant to begin dating in person again. “I’d chatted with some people on dating apps, but I wasn’t in a rush to meet up with anyone,” she says. “Everything about the pandemic had made me quite anxious.”</p><p>In early August, she agreed to meet someone from a dating app for a drink, her first date since March. “We’d been exchanging messages for a few months, and he was really nice,” says Emily, who did not want her full name used.</p><p>But when they did finally meet, she says, “I just felt extremely hesitant”. “In the back of my mind, I still wasn’t sure I was ready to date again. Later that day I sent him a text explaining how I felt, and he replied saying he had sensed that from my body language.”</p><p>Emily isn’t alone in feeling like dating amid Covid-19 is fraught. In fact, her behaviour chimes with a 2017 study in which a group of psychologists at Montréal’s McGill University looked into whether people’s dating behaviour would change if they were worried about the risk of infectious disease. Would people shy away from chasing romance if they were subconsciously aware of a potential health risk, or would the natural human desire to find a partner prevail?</p><p>The researchers had little idea that Covid-19 was around the corner. Now, their work, combined with other psychological studies conducted during the pandemic, offers a fascinating and highly relevant window into how the crisis appears to be affecting our dating behaviour. And, it points to ways in which we can date more effectively in the future as well as form deeper and stronger relationship bonds.</p><p>The McGill experiment suggests Emily’s avoidance might be down to an element of our psyche known as the “behavioural immune system”.</p><p>Pathogens have presented a threat to our survival throughout human history. So, evolutionary psychologists believe humans have evolved a set of subconscious responses that manifest when we are particularly concerned about the presence of an infectious disease. These responses lead us to engage in behavioural patterns that reduce the likelihood of getting infected, such as being less open and making reduced eye contact when in social situations.</p><p>The McGill team examined how this played out in a dating context. They took several hundred heterosexual male and female singletons, aged 18 to 35, and had them complete a known psychometric test known as PVDS, or the perceived vulnerability to disease scale. This consists of a 15-item questionnaire, asking participants to rate from 1 (strongly disagree) to 7 (strongly agree) their feelings on questions such as, ‘It really bothers me when people sneeze without covering their mouths’, or ‘My immune system protects me from most illnesses that other people get’.</p>",
        pubDate: "Wed, 25 Nov 2020 15:35:30 GMT",
        author: "David Cox",
        image: {
            _url: "https://ychef.files.bbci.co.uk/1600x900/p08ynxjp.webp",
            _height: "433",
            _width: "770"
        },
        timeStamp: "2020-11-25T15:35:30.000Z",
        displayDate: "November 25th 2020"
    },
    {
        slug: "news6",
        title: "The corporate ideals driving ‘secret parenting’",
        shortText: "Many parents downplay caring responsibilities at work to show commitment. Covid-19 has exposed the challenges that parents face – but will it change anything?",
        text: "<p>Six months after my daughter was born, I was back in the office, bleary eyed but eager to prove myself in a new position. A few weeks later, when I needed a few days off because of chickenpox at her day-care, I dreaded having to tell my team. Despite supportive colleagues, I felt intense pressure to act like nothing had changed.</p><p>Before, the job had always come first, something that had been key to getting ahead. Back at work as a mother, I was stressed because I couldn’t control my time, worried I now seemed more unreliable and anxious about the next time this might happen. I barely mentioned my daughter in the office; I would never have included an anecdote like this in my writing at that time.</p><p>Three years on, I realise it’s important to do so because this stress isn’t unique to me. Many people face the same pressure, because living up to workplace ideals is often not compatible with caring responsibilities, children or otherwise. This harms caregivers, who are statistically more likely to be women. Many of us respond by downplaying these responsibilities or convincing colleagues we can do just as much overtime as before, because we know that if we don’t, we risk falling victim to the numerous biases that hold mothers back.</p><p>Welcome to the world of “secret parenting”. Economist Emily Oster coined this relatable phrase in a 2019 article in which she urged parents to “come clean” about the nature of their lives. And while the pandemic – and our abrupt shift to home working – has forced many of our hands, it’s not clear whether unveiling the responsibilities we have outside the office will bring meaningful change.</p><p>Secret parenting can start as early as pregnancy. Research shows that some women hide their pregnancies, especially in roles in which they feel they have to compete with men and don’t want to reveal anything that might “get in the way” of work. Working women often feel the need to go “above and beyond” normal standards during pregnancy, another study showed, as well as hide any sickness for fear of appearing unreliable.</p><p>It continues after birth. Women report wanting to hide their breastfeeding at work because it is taboo, and doing so is such an obvious gendered difference that highlights their new status as mothers. There are plenty of anecdotes about women feeling forced to prove they can still work overtime and offering to work even when their children are ill as well as those who purposely don’t share photos of their children or talk about them at all.</p>",
        pubDate: "Mon, 23 Nov 2020 13:50:15 GMT",
        author: "Melissa Hogenboom",
        image: {
            _url: "https://ychef.files.bbci.co.uk/1600x900/p08ybylg.webp",
            _height: "433",
            _width: "770"
        },
        timeStamp: "2020-11-23T13:50:15.000Z",
        displayDate: "November 23rd 2020"
    }
];

describe("News5 tests", () => {
    it("renders correctly", () => {
        const {container} = render(<News5 newsItem={newsItem}/>);
        expect(container).toMatchSnapshot();
    });
});