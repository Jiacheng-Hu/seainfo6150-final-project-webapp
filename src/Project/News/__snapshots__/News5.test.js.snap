// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`News5 tests renders correctly 1`] = `
<div>
  <article
    class="container"
  >
    <header>
      <h1>
        Coronavirus: Why dating feel so different now
        <br />
         
        <small>
          by 
          David Cox
        </small>
      </h1>
      <time
        datetime="2020-11-25T15:35:30.000Z"
      />
    </header>
    <div
      class="news"
    >
      <img
        class="news-image-item"
        src="https://ychef.files.bbci.co.uk/1600x900/p08ynxjp.webp"
      />
      <div>
        <p>
          Emily, a 29-year-old surveyor from London, says she’s always been something of an introvert. She’d dabbled in dating, but when the UK’s first lockdown restrictions ended in July, she was reluctant to begin dating in person again. “I’d chatted with some people on dating apps, but I wasn’t in a rush to meet up with anyone,” she says. “Everything about the pandemic had made me quite anxious.”
        </p>
        <p>
          In early August, she agreed to meet someone from a dating app for a drink, her first date since March. “We’d been exchanging messages for a few months, and he was really nice,” says Emily, who did not want her full name used.
        </p>
        <p>
          But when they did finally meet, she says, “I just felt extremely hesitant”. “In the back of my mind, I still wasn’t sure I was ready to date again. Later that day I sent him a text explaining how I felt, and he replied saying he had sensed that from my body language.”
        </p>
        <p>
          Emily isn’t alone in feeling like dating amid Covid-19 is fraught. In fact, her behaviour chimes with a 2017 study in which a group of psychologists at Montréal’s McGill University looked into whether people’s dating behaviour would change if they were worried about the risk of infectious disease. Would people shy away from chasing romance if they were subconsciously aware of a potential health risk, or would the natural human desire to find a partner prevail?
        </p>
        <p>
          The researchers had little idea that Covid-19 was around the corner. Now, their work, combined with other psychological studies conducted during the pandemic, offers a fascinating and highly relevant window into how the crisis appears to be affecting our dating behaviour. And, it points to ways in which we can date more effectively in the future as well as form deeper and stronger relationship bonds.
        </p>
        <p>
          The McGill experiment suggests Emily’s avoidance might be down to an element of our psyche known as the “behavioural immune system”.
        </p>
        <p>
          Pathogens have presented a threat to our survival throughout human history. So, evolutionary psychologists believe humans have evolved a set of subconscious responses that manifest when we are particularly concerned about the presence of an infectious disease. These responses lead us to engage in behavioural patterns that reduce the likelihood of getting infected, such as being less open and making reduced eye contact when in social situations.
        </p>
        <p>
          The McGill team examined how this played out in a dating context. They took several hundred heterosexual male and female singletons, aged 18 to 35, and had them complete a known psychometric test known as PVDS, or the perceived vulnerability to disease scale. This consists of a 15-item questionnaire, asking participants to rate from 1 (strongly disagree) to 7 (strongly agree) their feelings on questions such as, ‘It really bothers me when people sneeze without covering their mouths’, or ‘My immune system protects me from most illnesses that other people get’.
        </p>
      </div>
    </div>
  </article>
</div>
`;
